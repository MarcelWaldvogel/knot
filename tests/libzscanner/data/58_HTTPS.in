$ORIGIN	.
$TTL	1

; OK
@	HTTPS	1	.

; KO
;@	SMIMEA

