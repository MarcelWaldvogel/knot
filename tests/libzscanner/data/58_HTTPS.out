OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0041
RDATA=000100
------
